arch-installer.sh

```diff
@@ -127,6 +127,11 @@ trap '
 # clean up after ourselves no matter how we die.
 trap 'exit 1;' SIGINT
 
+if [[ ! -e "$1" ]] && [[ "$1" == /dev/* ]]; then
+    echo "Couldn't find block device $1" >&2
+    exit 1
+fi
+
```

Comment:
Doing the block check after everything also helps us to ensure that `losetup` worked (in the other codepath).